modules/pipeline/providers/cron/cron.service.go

```diff
@@ -229,3 +231,249 @@ func (s *provider) CronUpdate(ctx context.Context, req *pb.CronUpdateRequest) (*
 
 	return &pb.CronUpdateResponse{}, nil
 }
+
+// edge
+
+func (s *provider) EdgeSyncCron(ctx context.Context, req *pb.EdgeCronSyncRequest) (*pb.EdgeCronSyncResponse, error) {
```

Comment:
no need this method.

Reply:
the only changes is: the request of `CronCreate` should assign `isEdge` correctly.
Central PIPELINE should assign it to true if necessary, and Edge PIPELINE just handle it correctly.